TTYC.Application/Users/Queries/ValidatePassword/ValidatePasswordQuery.cs

```diff
@@ -0,0 +1,10 @@
+﻿using MediatR;
+using TTYC.Domain;
+
+namespace TTYC.Application.Users.Queries.ValidatePassword
+{
+    public class ValidatePasswordQuery : IRequest<User>
```

Comment:
You validate a password without passing a password. It looks more like GetUserQuery to me